codalab/lib/bundle_cli.py

```diff
@@ -1189,6 +1189,17 @@ def do_workers_command(self, args):
                 help='Name of file containing patterns matching files and directories to exclude from upload. '
                 'This option is currently only supported with the GNU tar library.',
             ),
+            Commands.Argument(
+                '-l',
+                '--link',
+                help='Makes the path the source of truth of the bundle, meaning that the server will retrieve the '
+                'bundle directly from the specified path rather than storing its contents'
+                'in its own bundle store. If this argument is used and a local file path is specified,'
+                'the path must match a glob expression specified by the CODALAB_LINK_ALLOWED_PATHS environment variable'
+                'or the --link_allowed_paths command-line parameter to codalab_service.py.',
```

Comment:
Don't include the allowed path comments because this is on the server side, and the user won't be able to control these things anyway.